LiskHQ/lisk-hub
# Implement the list of transactions of the block on the block details page - closes #2506

### What issue have I solved?

-- #2506 

### How have I implemented/fixed it?

This PR contains the implementation of the Block Details page for Monitor, the code implemented are.

- [x] Add list of transactions for the block.
- [x] Add style for medium breakpoint.
- [x] Add HOC for provide prop (true/false) when the user resize the window.
- [x] Add unit test for blockDetails and HOC component.

### How has this been tested?

1. Run the app.
2. Type in the URL http://localhost:8080/#/monitor/blocks
3. Select any block.
4. Check the block details information.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
